# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/test/browser/setFetchCookie.sjs
# Commit: 98848e63dfbf
# Full Hash: 98848e63dfbf5702726d3e07fb92855263312fed
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2025-01-09 04:16:57
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 6: Add tests. r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223386
# ==============================================================================

diff -r b668d0c685e8 -r 98848e63dfbf netwerk/cookie/test/browser/setFetchCookie.sjs
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/netwerk/cookie/test/browser/setFetchCookie.sjs	Wed Jan 08 19:26:00 2025 +0000
@@ -0,0 +1,15 @@
+function handleRequest(request, response) {
+  response.setStatusLine(request.httpVersion, 200, "OK");
+  response.setHeader(
+    "Set-Cookie",
+    "thirdPartyFetch=value; SameSite=None; Secure;",
+    false
+  );
+  response.setHeader(
+    "Access-Control-Allow-Origin",
+    "https://example.com",
+    false
+  );
+  response.setHeader("Access-Control-Allow-Credentials", "true", false);
+  response.setHeader("Access-Control-Allow-Methods", "GET", false);
+}
